test/src/unit-allocator.cpp

```diff
@@ -96,6 +100,8 @@ struct my_allocator : std::allocator<T>
     {
         if (next_deallocate_fails)
         {
+            (void)p; // Ignored unused
```

Comment:
Why is this needed?  It's used in the `else` branch.

Reply:
@gregmarr MSVC 2015 does warn here. I presume it first uses the fact that `next_deallocate_fails == true` ditches the else part and then complains about `p` not being used.

Reply:
That's very strange, because `next_deallocate_fails` is only ever `false`.  Ah well, fun with MSVC.